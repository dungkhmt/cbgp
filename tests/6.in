30 30
30 30
27 3
14 1
28 2
17 1
19 1
5 0
19 10
17 5
25 0
8 16
27 1
8 22
2 4
18 29
25 26
21 1
24 11
27 16
9 16
7 4
6 0
28 29
12 11
21 13
28 23
25 11
18 20
4 0
15 28
5 1
1
5 1 21 1 0 0 14 8 1 1 13 3 4 2 2 2 15 5 14 4 26 3 10 4 12 4 27 1 29 9 29 1 16 4 13 1 35 0 24 3 38 0 25 1 13 5 28 3 11 5 8 4 12 8 18 4 31 0 33 0 