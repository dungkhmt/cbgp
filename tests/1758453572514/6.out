(27, 3), (14, 1), (28, 2), (17, 1), (19, 1), (5, 0), (19, 10), (17, 5), (25, 0), (8, 16), (27, 1), (8, 22), (2, 4), (18, 29), (25, 26), (21, 1), (24, 11), (27, 16), (9, 16), (7, 4), (6, 0), (28, 29), (12, 11), (21, 13), (28, 23), (25, 11), (18, 20), (4, 0), (15, 28), (5, 1), 
numberIntersection=0 
minDistance=5.0 
minAngle=0.9719297198828905 
sumAngle=33.86716613743765 
sumDistance=38236.4139646291 
minEdgeLength=5.0 
sumEdgeLength=685.1964020113934 

0.0000000000 5.0000000000 0.9719297199 33.8671661374 38236.4139646291 5.0000000000 685.1964020114
0: (108, 7),
1: (106, 37),
2: (120, 12),
3: (0, 76),
4: (120, 7),
5: (108, 30),
6: (108, 0),
7: (120, 0),
8: (82, 14),
9: (98, 21),
10: (101, 62),
11: (98, 14),
12: (103, 18),
13: (115, 46),
14: (0, 82),
15: (111, 60),
16: (93, 23),
17: (113, 35),
18: (120, 115),
19: (104, 47),
20: (120, 120),
21: (111, 42),
22: (76, 9),
23: (0, 120),
24: (93, 16),
25: (99, 7),
26: (75, 0),
27: (92, 28),
28: (120, 66),
29: (120, 110),
108 7 106 37 120 12 0 76 120 7 108 30 108 0 120 0 82 14 98 21 101 62 98 14 103 18 115 46 0 82 111 60 93 23 113 35 120 115 104 47 120 120 111 42 76 9 0 120 93 16 99 7 75 0 92 28 120 66 120 110 
0: (108, 7), 1: (106, 37), 2: (120, 12), 3: (0, 76), 4: (120, 7), 5: (108, 30), 6: (108, 0), 7: (120, 0), 8: (82, 14), 9: (98, 21), 10: (101, 62), 11: (98, 14), 12: (103, 18), 13: (115, 46), 14: (0, 82), 15: (111, 60), 16: (93, 23), 17: (113, 35), 18: (120, 115), 19: (104, 47), 20: (120, 120), 21: (111, 42), 22: (76, 9), 23: (0, 120), 24: (93, 16), 25: (99, 7), 26: (75, 0), 27: (92, 28), 28: (120, 66), 29: (120, 110), 
numberIntersection=0 minAngle=0.9719297198828905 sumAngle=33.867166138056355 minEdgeLength=5.0 sumEdgeLength=685.1964020114342 minDistance=5.0 sumDistance=38236.413964615465 
Time taken: 55.148 s
